templates/page.tpl.php

```diff
@@ -87,18 +87,15 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
-    <div class="wrapper">
-      <section class="footer-top">
-        <nav>
-          <?php print render($page['footer']); ?>
-        </nav>
-      </section>
-    </div>
-  </section>
-  <section class="page-bottom">
-    <div class="wrapper">
+  <div class="wrapper">
+    <section class="footer-top">
+      <?php print render($page['footer_top']); ?>
+    </section>
+    <section class="footer-bottom">
+      <?php print render($page['footer_bottom']); ?>
```

Comment:
That's getting pretty close to a content issue IMO.  I wonder if the UI Kit has a utility class that you can apply to a UL to render the list itms inline?  Might be worth asking in Slack.  If it did, then you could perhaps apply that to a menu block and the issue would be resolved.

If not, maybe we look at adding a utility class list that for the same purpose.

As an aside, the government crest guidelines state that the crest must be at the top of the page, not at the bottom.